src/overview/container.js

```diff
@@ -11,6 +11,10 @@ import ResultList from './components/ResultList'
 import Overview from './components/Overview'
 import PageResultItem from './components/PageResultItem'
 import ResultsMessage from './components/ResultsMessage'
+import Tags from 'src/common-ui/components/Tags'
+import TagOption from 'src/common-ui/components/TagOption'
+import NoResult from 'src/common-ui/components/NoResult'
```

Comment:
Export all of these from `src/common-ui/components/index.js`, then you can just add the wanted components to the single entrypoint import (L7)